.github/workflows/job_build_linux.yml

```diff
@@ -25,7 +25,7 @@ on:
       arch:
         description: 'Target architecture'
         type: string
-        default: 'x86_64'
+        default: 'intel64'
```

Comment:
why do you use intel64 instead of x86_64 ? we typically use x86_64  in OpenVINO

Reply:
I wanted to align it with artifacts storing action: https://github.com/openvinotoolkit/openvino/blob/c9288cce97bcf5e6337449768ec0c71492ce0621/.github/workflows/job_build_linux.yml#L124
@akladiev, can we use x86_64 for the `target_arch` input? Granted we updated the enum in https://github.com/openvinotoolkit/openvino/blob/c9288cce97bcf5e6337449768ec0c71492ce0621/.github/actions/common/constants.py#L9

Reply:
It's target arch from this path in OV artifacts:
![image](https://github.com/user-attachments/assets/4249c8a9-d4ff-46f1-ad98-737c4db96e35)
In internal validation this value is used at least in fuzzing tests where you need to provide a full path to libopenvino.so to test (input for-object option for llvm-cov tool), maybe also somewhere else

Reply:
intel64 is used within package
x86_64 is used as package name, arch names